about.html

```diff
@@ -0,0 +1,118 @@
+        <div class="contact">
+            <p>Please contact us per Email for any further questions about the Web Dev Program!</p>
+            <a href="#">example@email.com</a>
+        </div>
+    </section>
+
+
+    <section id="logo-section">
+        <div class="section-title"><h2>Microverse Logo</h2></div>
+        <span class="red-bar"></span>
+        <div><p>We provide some design guidelines for you to create the website, but you must personalize the content, instead of a conference you can build a website for a concert or for a web development course. It's very important the you personalize your project so that you have something unique in your portfolio to share with potential employers during job searching.</p></div>
+        <div id="logo-section-box"><img src="images/Microverse.png" alt="esverocim logo"></div>
+        <span class="gray-line"></span>
+    </section>
+
+
+    <section id="past-classes">
+        <div class="section-title"><h2>Microverse Logo</h2></div>
+        <span class="red-bar"></span>
+        <div class="past-classes-info"><p>Take a look at the last classes we done.</p></div>
+        <ul class="past-classes-container">
+            <li class="past-classes-box">
+                <a href="#"><img src="images/class-1.jpg" alt="computer science class full of students"></a>
+                <p>2022<br>Microverse</p>
+            </li>
+            <li class="past-classes-box">
+                <a href="#"><img src="images/class-2.webp" alt="computer science class full of students"></a>
+                <p>2022<br>Capstone Project</p>
+            </li>
+        </ul>
+    </section>
+
+
+    <section id="partner-section">
+        <div class="section-title"><h2>Partners</h2></div>
+        <span class="red-bar"></span>
+        <div class="partner-container">
+            <a href="#" class="partner-logo"><img src="images/google.png" alt="microverse logo"></a>
+            <a href="#" class="partner-logo"><img src="images/mezilla.svg" alt="microverse logo"></a>
+            <a href="#" class="partner-logo"><img src="images/NOVER.svg" alt="microverse logo"></a>
+            <a href="#" class="partner-logo"><img src="images/CodeClub.svg" alt="microverse logo"></a>
+            <a href="#" class="partner-logo"><img src="images/daushCamp.svg" alt="microverse logo"></a>
+        </div>
+    </section>
+
+
+    <footer id="footer">
+        <a href="#" class="logo-footer"><img src="images/Microverse.png" alt="microverse logo"></a>
+        <div class="footer-info-box">
+            <p id="footer-info">Est. 2021.<br>No Rights Reserved.</p>
+        </div>
+    </footer>
```

Comment:
# **Desktop size: About page**
- Make sure It has a dark background.
- Please this page, in desktop size does not contain the Partner section.
- See Your design:
![Capture](https://user-images.githubusercontent.com/63972670/210218600-2caa2ee9-1222-491e-8fc7-805037371758.PNG)
-
Look at the original design:

![Capture1](https://user-images.githubusercontent.com/63972670/210218682-e6cc7b6b-795b-4238-ac92-f6f92578b1e4.PNG)